apps/site/src/app/admin/applicants/components/ApplicantReviewerIndicator.tsx

```diff
@@ -0,0 +1,58 @@
+import {
+	Box,
+	SpaceBetween,
+	StatusIndicator,
+} from "@cloudscape-design/components";
```

Comment:
Note: I'm not sure if it actually matters with the transpiler in Next.js, but it might be safer to import these directly, see guide on [Using Cloudscape components](https://cloudscape.design/get-started/integration/using-cloudscape-components/).

Reply:
And same in `HackerApplicantActions.tsx`